spark/src/main/java/org/apache/iceberg/spark/actions/BaseSparkAction.java

```diff
@@ -109,40 +110,6 @@ protected JobGroupInfo newJobGroupInfo(String groupId, String desc) {
     return new JobGroupInfo(groupId + "-" + JOB_COUNTER.incrementAndGet(), desc, false);
   }
 
-  /**
-   * Returns all the path locations of all Manifest Lists for a given list of snapshots
-   * @param snapshots snapshots
-   * @return the paths of the Manifest Lists
-   */
-  private List<String> getManifestListPaths(Iterable<Snapshot> snapshots) {
```

Comment:
Can we remove the same method in class `BaseAction`? This applies to `getOtherMetadataFilePaths` as well.

Reply:
Yeah, we should clean up `BaseAction`. I am fine doing that in a follow-up PR too. @karuppayya, could you submit one when the current is in?